src/main/java/meteordevelopment/meteorclient/utils/render/color/Color.java

```diff
@@ -10,7 +10,36 @@
 import net.minecraft.nbt.NbtCompound;
 import net.minecraft.util.math.Vec3d;
 
+@SuppressWarnings("unused")
 public class Color implements ICopyable<Color>, ISerializable<Color> {
+
+    public static final Color white = new Color(java.awt.Color.WHITE);
+    public static final Color WHITE = white;
+    public static final Color lightGray = new Color(java.awt.Color.LIGHT_GRAY);
+    public static final Color LIGHT_GRAY = lightGray;
```

Comment:
is there a reason why there are two variables for each color?

Reply:
It's to follow along the java.awt format

Reply:
Leave just the uppercase ones.

Reply:
Done in e6dd40f

Reply:
If you're fine with how it looks, just squash merge it